highlighter/__init__.py

```diff
@@ -33,7 +33,7 @@ def markup_text(text):
         @:return marked text, e.g., <mark>highlighted text</mark>."""
         result = text
 
-        # TODO: add an implementation
+        result = "<mark>" + text + "</mark>"
```

Comment:
Why do you need it? Is it used anywhere?

Reply:
you are right may be needs to be removed